Automattic/delphin
# Fix wrong line endings

This pull request fixes line endings for two files that weren't using Unix line endings. This will reduce potential issues while deploying in production with Subversion.
#### Testing instructions
1. Run `git checkout fix/line-endings` and start your server, or open a [live branch](https://delphin.live/?branch=fix/line-endings)
2. Open the [`Home` page](http://delphin.localhost:1337/)
3. Check that everything works as before
#### Additional notes

You will have to rely on a more powerful diff tool than the one from GitHub since the latter just thinks the whole files changed.
#### Reviews

@Automattic/sdev-feed
